clients/drcachesim/simulator/simulator.cpp

```diff
@@ -338,6 +339,7 @@ simulator_t::print_core(int core) const
             }
         }
         std::cerr << ")" << std::endl;
+        return need_comma;
```

Comment:
This is subtle enough I think it would benefit from a comment.  Took me a bit to figure out why commas had anything to do with whether or not this core was active.

Reply:
Done.